jobs/resend_invitation_email/worker.go

```diff
@@ -62,21 +65,21 @@ func (rseworker *ResendInvitationEmailWorker) JobChannel() chan<- model.Job {
 
 func (rseworker *ResendInvitationEmailWorker) cleanEmailData(emailStringData string) []string {
 	// emailStringData looks like this ["user1@gmail.com","user2@gmail.com"]
-	t := strings.Trim(strings.Trim(emailStringData, "["), "]")
-	u := strings.Split(t, ",")
-	var cleaned []string
-	for _, ss := range u {
-		cleaned = append(cleaned, ss[1:len(ss)-1])
+	emails := []string{}
+	err := json.Unmarshal([]byte(emailStringData), &emails)
+	if err != nil {
+		log.Fatal(err)
```

Comment:
Probably we want to return an error in this function instead of just login Fatal